librz/core/cmd_api.c

```diff
@@ -274,7 +274,10 @@ static bool is_valid_argv_modes(RzCmdDesc *cd, char last_letter) {
 		return false;
 	}
 	char suffix[] = { last_letter, '\0' };
-	return cd->d.argv_modes_data.modes & suffix2mode(suffix);
+	RzOutputMode mode = rz_char_to_output_mode(suffix);
```

Comment:
Why this required changes?

Reply:
Initially, when `suffix2mode` doesn't match the string with any string in `argv_modes[]`, it returns 0. `is_valid_argv_modes` would then calculate the bitwise & with 0 to return 0. Since, I changed `return 0` to `return *suffix` in `suffix2mode`, I thought I would have a check in `is_valid_argv_modes` which returns 0 whenever `*suffix` is returned.